fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeAlterTableStatementTest.java

```diff
@@ -141,4 +141,10 @@ public void testRollup() {
     public void testAlterTableComment() {
         analyzeSuccess("alter table t0 comment = \"new comment\"");
     }
+
+    @Test
+    public void testAlterWithTimeType() {
+        analyzeFail("alter table t0 add column testcol TIME");
+        analyzeFail("alter table t0 modify column v0 TIME");
```

Comment:
Why not add a `create table` test case to cover creating table with TIME type.

Reply:
create table test case have existed in the code. We just miss for DDL